### PR TITLE
Add image alt attributes

### LINKED ISSUE #89: Add alt attributes to images as appropriate
We recently turned on template linting for this project but in an effort to get the PR merged we temporarily turned off the rule `img-alt-attributes` https://github.com/ember-learn/ember-website/blob/master/.template-lintrc.js#L6

We should turn that rule back on and add alt attributes to all images 👍

### PR DESCRIPTION
**Pull request by abhilashlr**
Fixes #89

### CODE REVIEW

File: app/templates/legal.hbs [hunk 1]
```diff
@@ -36,7 +36,11 @@
   <p>
     The graphs, diagrams and charts in the <a href="/guides">Ember Guides</a> have been released under the Creative Commons Attribution-NonCommercial-ShareAlike (CC BY-NC-SA) license.
   </p>
-  <img class="pull-right" src="/images/legal/license-cc_by_nc_sa.png">
+  <img
+    alt="Legal license"
```
Comment (mansona) [inline app/templates/legal.hbs]: This should probably have the details of the licence in the alt text (CC, By, Non Commercial, Share Alike)

File: app/templates/logos.hbs [hunk 2]
```diff
@@ -18,15 +18,14 @@
...
 
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Ember-Dark.png"></li>
-    <li><img src="/images/brand/ember_Ember-Light.png"></li>
-    <li><img src="/images/brand/ember_Ember-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Ember-1c-Dark.png"></li>
+    <li><img alt="Ember" src="/images/brand/ember_Ember-Dark.png"></li>
+    <li><img alt="Ember" src="/images/brand/ember_Ember-Light.png"></li>
+    <li><img alt="Ember" src="/images/brand/ember_Ember-1c-Light.png"></li>
+    <li><img alt="Ember" src="/images/brand/ember_Ember-1c-Dark.png"></li>
```
Comment (mansona) [inline app/templates/logos.hbs]: Each of these alt texts should be different and descriptive of the difference in the logos

Comment (abhilashlr) [reply app/templates/logos.hbs]: Can you please help me with the text that needs to be added to the alt attributes? Or can I just go ahead and just add them based on the image name?

File: app/templates/legal.hbs (hunk 1 above)
Comment (locks) [reply app/templates/legal.hbs]:
```suggestion
     alt="CC, By, Non Commercial, Share Alike"
```

File: app/templates/logos.hbs [hunk 3]
```diff
@@ -18,15 +18,14 @@
...
-
 <section>
   <h3>Primary Logo</h3>
 
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Ember-Dark.png"></li>
-    <li><img src="/images/brand/ember_Ember-Light.png"></li>
-    <li><img src="/images/brand/ember_Ember-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Ember-1c-Dark.png"></li>
+    <li><img alt="Ember" src="/images/brand/ember_Ember-Dark.png"></li>
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
     <li><img alt="Ember logo dark background" src="/images/brand/ember_Ember-Dark.png"></li>
```

File: app/templates/logos.hbs [hunk 4]
```diff
@@ -18,15 +18,14 @@
...
+    <li><img alt="Ember" src="/images/brand/ember_Ember-Light.png"></li>
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
     <li><img alt="Ember logo white background" src="/images/brand/ember_Ember-Light.png"></li>
```

File: app/templates/logos.hbs [hunk 5]
```diff
@@ -18,15 +18,14 @@
...
+    <li><img alt="Ember" src="/images/brand/ember_Ember-1c-Light.png"></li>
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
     <li><img alt="Ember logo red background white font" src="/images/brand/ember_Ember-1c-Light.png"></li>
```

File: app/templates/logos.hbs (hunk 2 above)
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
     <li><img alt="Ember logo black font" src="/images/brand/ember_Ember-1c-Dark.png"></li>
```

File: app/templates/logos.hbs [hunk 6]
```diff
@@ -111,56 +114,93 @@
...
+    </li>
   </ul>
 
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Jobs-Dark.png"></li>
-    <li><img src="/images/brand/ember_Jobs-Light.png"></li>
-    <li><img src="/images/brand/ember_Jobs-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Jobs-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Jobs" src="/images/brand/ember_Jobs-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Jobs logo dark background" src="/images/brand/ember_Jobs-Dark.png">
```

File: app/templates/logos.hbs [hunk 7]
```diff
@@ -111,56 +114,93 @@
...
+    </li>
+    <li>
+      <img alt="Ember Jobs" src="/images/brand/ember_Jobs-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Jobs logo light background" src="/images/brand/ember_Jobs-Light.png">
```

File: app/templates/logos.hbs [hunk 8]
```diff
@@ -111,56 +114,93 @@
...
+    </li>
+    <li>
+      <img alt="Ember Jobs" src="/images/brand/ember_Jobs-1c-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Jobs red background white font" src="/images/brand/ember_Jobs-1c-Light.png">
```

File: app/templates/logos.hbs [hunk 9]
```diff
@@ -111,56 +114,93 @@
...
+    </li>
+    <li>
+      <img alt="Ember Jobs" src="/images/brand/ember_Jobs-1c-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Jobs logo black font" src="/images/brand/ember_Jobs-1c-Dark.png">
```

File: app/templates/logos.hbs [hunk 10]
```diff
@@ -111,56 +114,93 @@
...
+      <img alt="Ember Times" src="/images/brand/ember_Times-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember Times" src="/images/brand/ember_Times-Light.png">
+    </li>
+    <li>
+      <img alt="Ember Times" src="/images/brand/ember_Times-1c-Light.png">
+    </li>
+    <li>
+      <img alt="Ember Times" src="/images/brand/ember_Times-1c-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Times black font" src="/images/brand/ember_Times-1c-Dark.png">
```

File: app/templates/logos.hbs [hunk 11]
```diff
@@ -111,56 +114,93 @@
...
-    <li><img src="/images/brand/ember_Times-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Times-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Times" src="/images/brand/ember_Times-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember Times" src="/images/brand/ember_Times-Light.png">
+    </li>
+    <li>
+      <img alt="Ember Times" src="/images/brand/ember_Times-1c-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Times red background white font" src="/images/brand/ember_Times-1c-Light.png">
```

File: app/templates/logos.hbs [hunk 12]
```diff
@@ -111,56 +114,93 @@
...
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Times-Dark.png"></li>
-    <li><img src="/images/brand/ember_Times-Light.png"></li>
-    <li><img src="/images/brand/ember_Times-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Times-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Times" src="/images/brand/ember_Times-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember Times" src="/images/brand/ember_Times-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Times logo light background" src="/images/brand/ember_Times-Light.png">
```

File: app/templates/logos.hbs [hunk 13]
```diff
@@ -111,56 +114,93 @@
...
+    </li>
   </ul>
 
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Times-Dark.png"></li>
-    <li><img src="/images/brand/ember_Times-Light.png"></li>
-    <li><img src="/images/brand/ember_Times-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Times-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Times" src="/images/brand/ember_Times-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Times logo dark background" src="/images/brand/ember_Times-Dark.png">
```

File: app/templates/logos.hbs [hunk 14]
```diff
@@ -111,56 +114,93 @@
...
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-Light.png">
+    </li>
+    <li>
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-1c-Light.png">
+    </li>
+    <li>
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-1c-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Forum logo black font" src="/images/brand/ember_Forum-1c-Dark.png">
```

File: app/templates/logos.hbs [hunk 15]
```diff
@@ -111,56 +114,93 @@
...
-    <li><img src="/images/brand/ember_Forum-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Forum-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-Light.png">
+    </li>
+    <li>
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-1c-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Forum logo red background white font" src="/images/brand/ember_Forum-1c-Light.png">
```

File: app/templates/logos.hbs [hunk 16]
```diff
@@ -111,56 +114,93 @@
...
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Forum-Dark.png"></li>
-    <li><img src="/images/brand/ember_Forum-Light.png"></li>
-    <li><img src="/images/brand/ember_Forum-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Forum-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Forum logo light background" src="/images/brand/ember_Forum-Light.png">
```

File: app/templates/logos.hbs [hunk 17]
```diff
@@ -111,56 +114,93 @@
...
 <section>
   <h3>Tertiary Logos</h3>
 
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Forum-Dark.png"></li>
-    <li><img src="/images/brand/ember_Forum-Light.png"></li>
-    <li><img src="/images/brand/ember_Forum-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Forum-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Forum" src="/images/brand/ember_Forum-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Forum logo dark background" src="/images/brand/ember_Forum-Dark.png">
```

File: app/templates/logos.hbs [hunk 18]
```diff
@@ -111,56 +114,93 @@
...
+      <img alt="Ember Data" src="/images/brand/ember_Data-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember Data" src="/images/brand/ember_Data-Light.png">
+    </li>
+    <li>
+      <img alt="Ember Data" src="/images/brand/ember_Data-1c-Light.png">
+    </li>
+    <li>
+      <img alt="Ember Data" src="/images/brand/ember_Data-1c-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Data logo black font" src="/images/brand/ember_Data-1c-Dark.png">
```

File: app/templates/logos.hbs [hunk 19]
```diff
@@ -111,56 +114,93 @@
...
-    <li><img src="/images/brand/ember_Data-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Data-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Data" src="/images/brand/ember_Data-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember Data" src="/images/brand/ember_Data-Light.png">
+    </li>
+    <li>
+      <img alt="Ember Data" src="/images/brand/ember_Data-1c-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Data logo red background white font" src="/images/brand/ember_Data-1c-Light.png">
```

File: app/templates/logos.hbs [hunk 20]
```diff
@@ -111,56 +114,93 @@
...
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Data-Dark.png"></li>
-    <li><img src="/images/brand/ember_Data-Light.png"></li>
-    <li><img src="/images/brand/ember_Data-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Data-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Data" src="/images/brand/ember_Data-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember Data" src="/images/brand/ember_Data-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Data logo light background" src="/images/brand/ember_Data-Light.png">
```

File: app/templates/logos.hbs [hunk 21]
```diff
@@ -111,56 +114,93 @@
...
+    </li>
   </ul>
 
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Data-Dark.png"></li>
-    <li><img src="/images/brand/ember_Data-Light.png"></li>
-    <li><img src="/images/brand/ember_Data-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_Data-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember Data" src="/images/brand/ember_Data-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember Data logo dark background" src="/images/brand/ember_Data-Dark.png">
```

File: app/templates/logos.hbs [hunk 22]
```diff
@@ -111,56 +114,93 @@
...
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-Light.png">
+    </li>
+    <li>
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-1c-Light.png">
+    </li>
+    <li>
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-1c-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember CLI black font" src="/images/brand/ember_CLI-1c-Dark.png">
```

File: app/templates/logos.hbs [hunk 23]
```diff
@@ -111,56 +114,93 @@
...
-    <li><img src="/images/brand/ember_CLI-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_CLI-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-Light.png">
+    </li>
+    <li>
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-1c-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember CLI logo red background white font" src="/images/brand/ember_CLI-1c-Light.png">
```

File: app/templates/logos.hbs [hunk 24]
```diff
@@ -111,56 +114,93 @@
...
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_CLI-Dark.png"></li>
-    <li><img src="/images/brand/ember_CLI-Light.png"></li>
-    <li><img src="/images/brand/ember_CLI-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_CLI-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-Dark.png">
+    </li>
+    <li>
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-Light.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember CLI logo light background" src="/images/brand/ember_CLI-Light.png">
```

File: app/templates/logos.hbs [hunk 25]
```diff
@@ -111,56 +114,93 @@
...
 <section>
   <h3>Subproject Logos</h3>
 
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_CLI-Dark.png"></li>
-    <li><img src="/images/brand/ember_CLI-Light.png"></li>
-    <li><img src="/images/brand/ember_CLI-1c-Light.png"></li>
-    <li><img src="/images/brand/ember_CLI-1c-Dark.png"></li>
+    <li>
+      <img alt="Ember CLI" src="/images/brand/ember_CLI-Dark.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
       <img alt="Ember CLI logo dark background" src="/images/brand/ember_CLI-Dark.png">
```

File: app/templates/logos.hbs [hunk 26]
```diff
@@ -36,72 +35,76 @@
...
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-Black-Rounded-Rectangle.png">
+          <img
+            alt="Ember icon"
+            src="/images/brand/ember_E-Icon-1c-Black-Rounded-Rectangle.png"
+          >
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-Black-Circle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-1c-Black-Circle.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons black circle" src="/images/brand/ember_E-Icon-1c-Black-Circle.png">
```

File: app/templates/logos.hbs [hunk 27]
```diff
@@ -36,72 +35,76 @@
...
     <li>
       <ul>
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-Black-Square.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-1c-Black-Square.png">
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-Black-Rounded-Rectangle.png">
+          <img
+            alt="Ember icon"
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
             alt="Ember icons black rounded rectangle"
```

File: app/templates/logos.hbs [hunk 28]
```diff
@@ -36,72 +35,76 @@
...
-          <img src="/images/brand/ember_E-Icon-1c-White-Circle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-1c-White-Circle.png">
         </li>
       </ul>
     </li>
     <li>
       <ul>
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-Black-Square.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-1c-Black-Square.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons black square" src="/images/brand/ember_E-Icon-1c-Black-Square.png">
```

File: app/templates/logos.hbs [hunk 29]
```diff
@@ -36,72 +35,76 @@
...
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-White-Rounded-Rectangle.png">
+          <img
+            alt="Ember icon"
+            src="/images/brand/ember_E-Icon-1c-White-Rounded-Rectangle.png"
+          >
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-White-Circle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-1c-White-Circle.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons white circle" src="/images/brand/ember_E-Icon-1c-White-Circle.png">
```

File: app/templates/logos.hbs [hunk 30]
```diff
@@ -36,72 +35,76 @@
...
     <li>
       <ul>
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-White-Square.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-1c-White-Square.png">
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-White-Rounded-Rectangle.png">
+          <img
+            alt="Ember icon"
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
             alt="Ember icons white rounded rectangle"
```

File: app/templates/logos.hbs [hunk 31]
```diff
@@ -36,72 +35,76 @@
...
-          <img src="/images/brand/ember_E-Icon-4c-Circle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Circle.png">
         </li>
       </ul>
     </li>
     <li>
       <ul>
         <li>
-          <img src="/images/brand/ember_E-Icon-1c-White-Square.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-1c-White-Square.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons white square" src="/images/brand/ember_E-Icon-1c-White-Square.png">
```

File: app/templates/logos.hbs [hunk 32]
```diff
@@ -36,72 +35,76 @@
...
-          <img src="/images/brand/ember_E-Icon-4c-Square.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Square.png">
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Circle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Circle.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons circle light background" src="/images/brand/ember_E-Icon-4c-Circle.png">
```

File: app/templates/logos.hbs [hunk 33]
```diff
@@ -36,72 +35,76 @@
...
     </li>
     <li>
       <ul>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Square.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Square.png">
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons rounded rectangle light background" src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
```

File: app/templates/logos.hbs [hunk 34]
```diff
@@ -36,72 +35,76 @@
...
-          <img src="/images/brand/ember_E-Icon-4c-Circle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Circle.png">
         </li>
       </ul>
     </li>
     <li>
       <ul>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Square.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Square.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons square light background" src="/images/brand/ember_E-Icon-4c-Square.png">
```

File: app/templates/logos.hbs [hunk 35]
```diff
@@ -36,72 +35,76 @@
...
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Circle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Circle.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons circle dark background" src="/images/brand/ember_E-Icon-4c-Circle.png">
```

File: app/templates/logos.hbs [hunk 36]
```diff
@@ -36,72 +35,76 @@
...
   <ul class="brand-logos">
     <li>
       <ul>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Square.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Square.png">
         </li>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons rounded rectangle dark background" src="/images/brand/ember_E-Icon-4c-Rounded-Rectangle.png">
```

File: app/templates/logos.hbs [hunk 37]
```diff
@@ -36,72 +35,76 @@
...
-
 <section>
   <h3>Icon</h3>
 
   <ul class="brand-logos">
     <li>
       <ul>
         <li>
-          <img src="/images/brand/ember_E-Icon-4c-Square.png">
+          <img alt="Ember icon" src="/images/brand/ember_E-Icon-4c-Square.png">
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
           <img alt="Ember icons square dark background" src="/images/brand/ember_E-Icon-4c-Square.png">
```

File: app/templates/logos.hbs [hunk 38]
```diff
@@ -36,72 +35,76 @@
...
   <h3>Tomster Lockup</h3>
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
     <li><img alt="Tomster lockup transparent background" src="/images/brand/ember_Tomster-Lockup.png"></li>
```

File: app/templates/logos.hbs [hunk 39]
```diff
@@ -36,72 +35,76 @@
...
 <section>
   <h3>Tomster Lockup</h3>
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
     <li><img alt="Tomster lockup red background" src="/images/brand/ember_Tomster-Lockup.png"></li>
```

File: app/templates/logos.hbs [hunk 40]
```diff
@@ -36,72 +35,76 @@
...
-
 <section>
   <h3>Tomster Lockup</h3>
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
     <li><img alt="Tomster lockup light background" src="/images/brand/ember_Tomster-Lockup.png"></li>
```

File: app/templates/logos.hbs [hunk 41]
```diff
@@ -36,72 +35,76 @@
...
 
-
 <section>
   <h3>Tomster Lockup</h3>
   <ul class="brand-logos">
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img src="/images/brand/ember_Tomster-Lockup.png"></li>
+    <li><img alt="Tomster" src="/images/brand/ember_Tomster-Lockup.png"></li>
```
Comment (locks) [inline app/templates/logos.hbs]:
```suggestion
     <li><img alt="Tomster lockup dark background" src="/images/brand/ember_Tomster-Lockup.png"></li>
```

**Pull request closed without merging**